rasa/core/training/interactive.py

```diff
@@ -816,10 +816,15 @@ def _write_stories_to_file(
     else:
         append_write = "w"  # make a new file if not
 
+    with open(
```

Comment:
do you mind adding a test to check this will work in the future?

Reply:
@rafaelteodosio and i don't know how make the requested test. @tmbo could you give us some advices?

Reply:
Look here `tests/core/training/test_interactive.py`. You can write a test that exports twice and assert that the correct count is used for both.